tools/fuse_convert.py

```
@@ -390,20 +436,73 @@ def legacy_load_after(mod_folder):
+        clean(
+            {
+                "Id": reference.get("id"),
+                "NotBefore": reference.get("notBefore"),
+                "NotAfter": reference.get("notAfter"),
+            }
+        )
+        for reference in convert_conflict_references(conflicts)
+    ]
+
+
+def _mixinto_requirement_ids(value):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_mixinto_requirement_ids`

(ANN202)